packages/realm/src/Realm.ts

```diff
@@ -49,6 +49,9 @@ import {
   ClassHelpers,
   normalizeObjectSchema,
   toArrayBuffer,
+  RealmListeners,
+  RealmEventName,
```

Comment:
not sure if you have thought about this being called `RealmEventName`. I imagine `RealmListenerEventName` seems redundant/weird since listener listens to events